### PR TITLE
Spaces in separator

### PR DESCRIPTION
**Pull request by kkocel**
Resolves https://github.com/sirrodgepodge/morgan-body/issues/48

### CODE REVIEW

**Comment by sirrodgepodge** [comment]
Hey there @kkocel love the PRs!

I do think they'll break tests as the tests are checking exact console output, they're pretty useful, did you run them?

**Comment by kkocel** [comment]
@sirrodgepodge  Hey, I ran the test and found out that mocked time (`forceDateToZero.js`) does not work if the machine timezone differs from EST. However, I forced TZ by using `process.env.TZ` (which is not recommended btw - most people recommend using jest). 

After setting TZ all tests passed :)

File: test/util/forceDateToZero.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 // need to standardize date to standardize log tests
 var OrigDate = Date;
 var testTime = 0;
+process.env.TZ = 'EST';
```
Comment (VottusCode) [inline test/util/forceDateToZero.js:4]: Good, this was breaking the CI before. Will need to find a better solution, but this should be OK for now.

**Review by VottusCode — APPROVED** [review]
LGTM

**Comment by VottusCode** [comment]
The test fails for me. Could you please check it again?

**Comment by kkocel** [comment]
@VottusCode can you share logs? I fixed tests in: https://github.com/sirrodgepodge/morgan-body/pull/49/commits/bd5e9a921c4f84510d7d33a7dfdabde1f8b6143d

**Comment by kkocel** [comment]
@VottusCode I added another PR with tests in one commit: https://github.com/sirrodgepodge/morgan-body/pull/54/files

**Comment by VottusCode** [comment]
@kkocel Merged the new PR. That one passes. I appreciate the contribution!

**Comment by kkocel** [comment]
Do you know when it will be published?

**Pull request merged**